Dockerfile

```diff
@@ -1,6 +1,15 @@
-FROM golang:1.14
+FROM registry.redhat.io/rhel8/go-toolset:1.15 as builder
```

Comment:
can you push to 1.16

why change the base image to the one in redhat?

Reply:
Because it's the one I use daily for Che and CodeReady Workspaces (so I know how it works!), and because the runtime image switches to SCRATCH so it doesn't really matter if you use an alpine or Red Hat based builder image -- with the above Dockerfile you end up with a smaller image than if you used the Alpine based go builder and deployed directly into that container. 

But as I'm not a project owner, and this is just to unblock our use of the Mattermost notifier for https://mattermost.eclipse.org/eclipse/channels/eclipse-che-releases ... if you prefer to use Alpine w/ Golang 1.16, I can certainly change the PR to use that. :D

Reply:
im fine with that, just asking to understand why the change and to get more clarifications